@navikt/ds-react/accordion-menu/AccordionMenuItem.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { forwardRef } from "react";
+import { OverridableComponent } from "../util";
+import cl from "classnames";
+
+export type AccordionMenuItemType = OverridableComponent<AccordionMenuItemProps>;
+
+export interface AccordionMenuItemProps {
+  props: {
+    children: React.ReactNode;
+    active?: boolean;
+    ariaCurrent?: string;
+  } & React.HTMLAttributes<HTMLAnchorElement>;
+  defaultComponent: "a";
+}
+
+const AccordionMenuItem: AccordionMenuItemType = forwardRef(
+  (
+    {
+      children,
+      component: Component = "a",
+      active = false,
+      ariaCurrent = "page",
+      className,
+      ...rest
+    },
+    ref
+  ) => {
+    return (
+      <Component
+        ref={ref}
+        className={cl("navds-accordion-menu-item", className, {
+          "navds-accordion-menu-item--active": active,
+        })}
+        aria-current={active && ariaCurrent}
```

Comment:
Er det ikke rart å sette verdien for en html-prop på en komponent (`ariaCurrent="page"`), også blir det ikke propagert til elementet med mindre active er true?

Reply:
Jeg har egentlig ikke noe forslag til en bedre løsning selv, forresten. Men jeg lurer på det beste kanskje bare er å overlate det til bruker, og å dokumentere det tydelig.

Reply:
Ja tenker kanskje å overlate dette til bruker vil være en god ide.